code/__HELPERS/mobs.dm

```diff
@@ -54,8 +54,8 @@ proc/random_name(gender, speciesName = "Human")
 	if(S)
 		return S.makeName(gender)
 	else
-		var/datum/species/human/H
-		return H.makeName(gender)
+		return call(/datum/species/proc/makeName)(gender)

```

Comment:
is call cheaper than instantiating a datum? Couldn't you just have a globally scoped datum that new players can read from? I guess its only called once in a while outside of roundstart